packages/babel-traverse/src/scope/index.js

```diff
@@ -45,6 +45,20 @@ function gatherNodeParts(node: Object, parts: Array) {
   }
 }
 
+function isScopeInFunctionParameter(scope) {
+  //todo: check arrow function expression or any util method?
+  if (scope.parent && scope.parent.block.type === "FunctionDeclaration") {
+    const fdPath = scope.path.findParent(
+      path => path.type === "FunctionDeclaration",
+    );
+    if (fdPath) {
+      return !!fdPath.findParent(path => {
+        return fdPath.node.params.includes(path.node);
```

Comment:
I'm not sure that I understand this. If `path.node` must be _inside_ the parameters of `fdPath`, why are we searching it in the _parents_ of `fdPath`?